src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -18,105 +21,108 @@
 import com.revature.services.PersonLogic;
 import com.revature.services.PersonRoleLogic;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.services.PersonRoleLogic'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)